lib/engine/round/g_18_mex/operating.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative '../operating'
+
+module Engine
+  module Round
+    module G18MEX
+      class Operating < Operating
+        def house_keeping
```

Comment:
can't you just override start operating? i don't know if house keeping adds that much

Reply:
That I tried at first. If I do "house_keeping" + super, this means that the "XXX operating YYY" message comes after the mail message.
So I could solve that by overriding skip_steps to do mail + super.
Then I got the idea about 1822, and its housekeeping.

But sure, I can go for the override of skip_steps.